### PR TITLE
add component customer question and buid product

### PR DESCRIPTION
**Pull request by thanhbui27**

### CODE REVIEW

**Comment by thanhbui27** [comment]
image

File: src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+type TItemAccordion = {
+  item: TDataCustomerQuestion;
+};
+
+const ItemAccordion: React.FC<TItemAccordion> = ({ item }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleClick = () => {
+    setIsOpen(!isOpen);
```
Comment (baonguyen26) [inline src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx:12]: setIsOpen((isOpen) => !isOpen);

File: src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import { TDataCustomerQuestion } from "../../constants";
+
+type TItemAccordion = {
+  item: TDataCustomerQuestion;
+};
+
+const ItemAccordion: React.FC<TItemAccordion> = ({ item }) => {
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (baonguyen26) [inline src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx:9]: type of state

File: src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            aria-expanded="true"
+            aria-controls="collapseOne"
+          >
+            {item.title}
+            <span
+              className={`${
+                isOpen ? `rotate-[-180deg] -mr-1` : `rotate-0`
+              } ml-auto h-5 w-5 shrink-0 transition-transform duration-200 ease-in-out motion-reduce:transition-none `}
+            >
+              <svg
```
Comment (baonguyen26) [inline src/components/CustomerQuestion/components/AccordionQuestion/ItemAccordion.tsx:34]: define constant or icon for this one

**Review by baonguyen26 — APPROVED** [review]

**Pull request merged**